tensorboard/components/experimental/plugin_util/plugin-host-ipc.ts

```diff
@@ -13,11 +13,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 namespace tb_plugin.host {
+  export type PluginHostMessageContext = {
+    pluginName: string | null;
```

Comment:
leave a comment on when this can be a `null`.

Reply:
Done